### ISSUE TITLE
Novation Dicer to scroll [QUESTION]

### ISSUE THREAD

**Issue by Capitaine-Pete** [pideck/pideck#90]
Hello ! 
I just discovered your project and it rocks ! 
I saw that Novation Dicer is supported by xwax/PiDeck but I'm wondering if I can use the Dicer controller to scroll the library, select and load a track ?

**Comment by danielhjames** [comment]
Yes, this should be possible with a modification to the Dicer support in xwax. There are some unused modes in the current Dicer map which could be used for this purpose without disrupting current functionality. 
The best place to ask for details is the xwax-devel mailing list, or take a look at the code: https://github.com/xwax/xwax/blob/master/dicer.c
null

**Comment by Capitaine-Pete** [comment]
Great ! Thanks for your quick answer !

**Comment by Capitaine-Pete** [comment]
I found my answer here https://sourceforge.net/p/xwax/mailman/message/31902486/ and it seems to be very difficult to to.

PiDeck is working great with my M-Audio Conectiv soundcard and the Novation Dicer, very good latency, I'm just not comfortable with browsing in the library.  

In order to avoid to use the touchscreen, I would like to use a keyboard but is it possible to just use a USB Numeric Keypad and reconfigure the commands/keys ?

**Comment by danielhjames** [comment]
Hi @Capitaine-Pete you can certainly use an ordinary USB keyboard which will give you all the shortcuts shown in https://github.com/pideck/pideck/blob/master/keyboard_control.md plus text searching on the library using any letter or number keys. Personally I use a small USB keyboard without numeric keypad to save space. I'm interested to try a backlit version such as the Perixx Periboard-411 as this should be better in the dark :-)

If you want to change xwax so that the numeric keypad manipulates the library, that would be a cool hack! You will probably have to block the number keys from searching the library. Here is an example of how to modify interface.c to change the key bindings: https://github.com/xwax/xwax/commit/85c9eb89bea7d8695e0e49ec289e2165b29d46e6
